concent_api/api-integration-force-payment.py

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env python3
```

Comment:
Why is it in this pull request too ?

Reply:
Hmm probably because I've rebase this branch on the api-test one, with different pull-request #153
Should I split it?

Reply:
Not necessary, I just wanted to clarify.